packages/openid4vp/src/models/z-verifier-attestations.ts

```diff
@@ -0,0 +1,11 @@
+import z from 'zod'
+
+const zVerifierAttestation = z.object({
+  format: z.string(),
+  data: z.record(z.any()).or(z.string()),
+  credential_ids: z.array(z.string()).optional(),
+})
+
+export const zVerifierAttestations = z.array(zVerifierAttestation)
+
+export type VerfierAttestations = z.infer<typeof zVerifierAttestations>
```

Comment:
I think it might make more sense to export type `VerifierAttestation`, because if you need the array type it's always easy to do `VerifierAttestation[]`